packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -95,69 +100,93 @@ const StyledOpacityTransitionContainer = styled("div", {
 
 const StyledSideNav = styled("nav", {
   shouldForwardProp: (prop) =>
-    prop !== "odysseyDesignTokens" && prop !== "isSideNavCollapsed",
+    prop !== "backgroundColor" &&
+    prop !== "odysseyDesignTokens" &&
+    prop !== "isAppContentWhiteBackground" &&
+    prop !== "isSideNavCollapsed",
 })<{
-  odysseyDesignTokens: DesignTokens;
+  isAppContentWhiteBackground: boolean;
+  backgroundColor?: UiShellColors["sideNavBackgroundColor"];
   isSideNavCollapsed: boolean;
-}>(({ odysseyDesignTokens, isSideNavCollapsed }) => ({
-  position: "relative",
-  display: "inline-block",
-  height: "100%",
-  backgroundColor: odysseyDesignTokens.HueNeutralWhite,
-
-  "&::after": {
-    backgroundColor: odysseyDesignTokens.HueNeutral200,
-    content: "''",
+  odysseyDesignTokens: DesignTokens;
+}>(
+  ({
+    backgroundColor,
+    isAppContentWhiteBackground,
+    isSideNavCollapsed,
+    odysseyDesignTokens,
+  }) => ({
+    position: "relative",
+    display: "inline-block",
     height: "100%",
-    opacity: 0,
-    position: "absolute",
-    right: 0,
-    top: 0,
-    transform: `translateX(0)`,
-    transition: `opacity ${odysseyDesignTokens.TransitionDurationMain}, transform ${odysseyDesignTokens.TransitionDurationMain}`,
-    width: odysseyDesignTokens.Spacing2,
-    zIndex: 2,
-  },
+    backgroundColor: backgroundColor || odysseyDesignTokens.HueNeutralWhite,
 
-  "&:has([data-sidenav-toggle='true']:hover), &:has([data-sidenav-toggle='true']:focus-visible)":
-    {
-      ...(isSideNavCollapsed && {
-        "&::after": {
-          opacity: 1,
-          transform: `translateX(100%)`,
-        },
+    ...(isAppContentWhiteBackground && {
+      borderRight: `${odysseyDesignTokens.BorderWidthMain} ${odysseyDesignTokens.BorderStyleMain} ${odysseyDesignTokens.HueNeutral100}`,
```

Comment:
To make this easier to read, we can do:
```suggestion
      borderRightWidth: odysseyDesignTokens.BorderWidthMain;
      borderRightStyle: odysseyDesignTokens.BorderStyleMain;
      borderRightColor: odysseyDesignTokens.HueNeutral100;
```

Reply:
I can change this if we really want but, the shorthand version is pretty standard. Doesn't feel particularly difficult to read